### ISSUE TITLE
Error starting Webstorm with plugin active

### ISSUE THREAD

**Issue by sawa-ko** [pandier/intellij-discord-rp#102]
### What happened?

I just open Webstorm and the error appears. I have Discord previously opened when I do it.

```
Failed to connect

io.github.vyfor.kpresence.exception.ConnectionException: java.net.SocketException: No existe el archivo o el directorio
	at io.github.vyfor.kpresence.ipc.Connection$UnixConnection.open(Connection.kt:164)
	at io.github.vyfor.kpresence.ipc.Connection.open(Connection.kt:28)
	at io.github.vyfor.kpresence.RichClient.connect(RichClient.kt:55)
	at io.github.vyfor.kpresence.RichClient.connect$default(RichClient.kt:49)
	at io.github.pandier.intellijdiscordrp.service.DiscordServiceKt.connect(DiscordService.kt:41)
	at io.github.pandier.intellijdiscordrp.service.DiscordServiceKt.access$connect(DiscordService.kt:1)
	at io.github.pandier.intellijdiscordrp.service.DiscordService$reconnect$2.invokeSuspend(DiscordService.kt:106)
	at io.github.pandier.intellijdiscordrp.service.DiscordService$reconnect$2.invoke(DiscordService.kt)
	at io.github.pandier.intellijdiscordrp.service.DiscordService$reconnect$2.invoke(DiscordService.kt)
	at io.github.pandier.intellijdiscordrp.util.MergingRunner.run(MergingRunner.kt:31)
	at io.github.pandier.intellijdiscordrp.service.DiscordService.reconnect(DiscordService.kt:99)
	at io.github.pandier.intellijdiscordrp.service.DiscordService$reconnectBackground$1.invokeSuspend(DiscordService.kt:133)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.internal.SoftLimitedDispatcher$Worker.run(SoftLimitedDispatcher.kt:125)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:99)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
Caused by: java.net.SocketException: No existe el archivo o el directorio
	at java.base/sun.nio.ch.UnixDomainSockets.connect0(Native Method)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:166)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:162)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:878)
	at java.base/java.nio.channels.SocketChannel.open(SocketChannel.java:285)
	at io.github.vyfor.kpresence.ipc.Connection$UnixConnection.open(Connection.kt:158)
	... 19 more

```

### Expected behaviour

Connect to the rich presense of Discord without any problem. I am using `Ubuntu Desktop 24.04` and Discord `stable 339221 (3687267) Host 0.0.72 x64 Build Override: N/A Linux 64-bit (6.8.0-47-generic)`

### What version of the plugin are you using?

1.6.0

### What IDE and version are you using?

WebStorm 2024.2.4

**Comment by pandier** [comment]
How did you install Discord and Webstorm? (snap, flatpak)

**Comment by sawa-ko** [comment]
Discord snap and Webstorm using Jetbrains Toolbox

**Comment by pandier** [comment]
Can you check where your `discord-ipc-0` file is located (or if it even exists)? Potential candidates: `/run/user/1000/discord-ipc-0`, `/run/user/1000/snap.discord/discord-ipc-0`

**Comment by sawa-ko** [comment]
![image](https://github.com/user-attachments/assets/6dd58d61-84cc-4388-b992-a9234c01e4ed)

**Comment by pandier** [comment]
Fixed in `v1.6.1`, version will be available on JetBrains Marketplace in ~2 business days
